test/cypress/integration/widgets/filters_spec.ts

```diff
@@ -0,0 +1,116 @@
+import { FilterRule } from '../../../../widgets/common/src/filters/types';
+
+describe('Filters widget', () => {
+    before(() => {
+        cy.usePageMock('filters').activate().mockLogin();
+    });
+
+    const filterName = 'filters_test_filter';
+    const filterRules: FilterRule[] = [
+        { type: 'attribute', key: 'blueprint_id', values: ['bpid'], operator: 'any_of' },
+        { type: 'label', key: 'precious', values: ['yes'], operator: 'any_of' }
+    ];
+
+    beforeEach(() => {
+        cy.deleteDeploymentsFilters(filterName).createDeploymentsFilter(filterName, filterRules).refreshPage();
+        cy.get('input[placeholder="Search..."]').type(filterName);
+        cy.get('.loading').should('not.exist');
```

Comment:
Optoinal:

How about reusing https://github.com/cloudify-cosmo/cloudify-stage/blob/a95d27c7ac29d5589adf8ee2801da8a37015c4a0/test/cypress/support/commands.ts#L49-L52

```suggestion
        cy.waitUntilPageLoaded();
```

Reply:
Can't be done - `.loader` is the main, full screen spinner visible before the app is loaded, while here we are waiting for `.loading`, which is the small in-widget spinner.

Reply:
Oh, I see :slightly_smiling_face: Thanks, I did not notice the different class name :sweat_smile: